docker/Dockerfile

```diff
@@ -29,12 +29,14 @@ RUN apt-get update \
     # Clean up what can be cleaned up.
     && apt-get autoremove -y
 
+# Install uv
+COPY --from=ghcr.io/astral-sh/uv:0.4.13 /uv /bin/uv
```

Comment:
I think I tried all possible ways to extract 0.4.13 into a variable (with both ENV and ARG), and failed.